### ISSUE TITLE
Does this prevent the camera from being used on PS5?

### ISSUE THREAD

**Issue by conner-fallone** [Hackinside/PS5_camera_files#16]
I tried to do this unsuccessfully, and noticed that the camera is no longer working on my PS5. Is this expected? I got concerned when I saw it was uploading drivers to the camera, but it was too late. If so, there should be a disclaimer about this. Is there anyway I can get it working again?

edit: now it's malfunctioning everytime i connect to my computer and I can't even do anything with it :(

**Comment by Hackinside** [comment]
It is probably a faulty USB port. The firmware isn't permanently stored in the camera.

**Comment by fallXone** [comment]
> It is probably a faulty USB port. The firmware isn't permanently stored in the camera. It loses its firmware whenever unplugged.

I use this camera every night on my PS5 and it 100% was working before. It's not working in any port at the moment.

**Comment by Hackinside** [comment]
As I said, the software or firmware doesn't change anything on the PS5 camera. So, what could happen to your camera is a mystery.

**Comment by fallXone** [comment]
> As I said, the software or firmware doesn't change anything on the PS5 camera. So, what could happen to your camera is a mystery.

I believe you, just trying to figure out what happened here. Think something went bad inside the camera during all of this.. I'm trying to get it to go through successfully thinking it would maybe "fix" the camera.

**Comment by fallXone** [comment]
I'm at a loss. Will likely be buying a new camera. Would probably be a good idea to add a disclaimer to the readme. I cannot get it to work at all with PC or PS5 now.

**Comment by Hackinside** [comment]
As far as I know, that has never happened to anyone in the past. Even after four years of using the PS4 camera with this method, I can't remember anyone reporting this issue. 
Do the following method: 
Do power off the PS5, do not put it on rest mode. 
Then, connect the camera to the PS5. 
Unplug the PS5 power cord from the main socket, and press and hold for 10s the power button to drain its power.  
Reconnect the power cord and power on the PS5.
Test the camera again and let me know the results.

**Comment by Hackinside** [comment]
It should be connected to the blue back port of your PS5 
https://youtu.be/rleoTD8Wmwc

**Comment by fallXone** [comment]
> It should be connected to the blue back port of your PS5 https://youtu.be/rleoTD8Wmwc

yup i'm aware. both ports are not detecting my camera anymore. When i go to settings now it's greyed out.

**Comment by fallXone** [comment]
> As far as I know, that has never happened to anyone in the past. Even after four years of using the PS4 camera with this method, I can't remember anyone reporting this issue. Do the following method: Do power off the PS5, do not put it on rest mode. Then, connect the camera to the PS5. Unplug the PS5 power cord from the main socket, and press and hold for 10s the power button to drain its power. Reconnect the power cord and power on the PS5. Test the camera again and let me know the results.

No luck with this either sadly.
